cav_msgs/msg/MobilityHeader.msg

```diff
@@ -12,9 +12,8 @@ string  sender_id
 # Empty string indicates a broadcast message
 string  recipient_id
 
-# sender's dynamic ID which is its BSM id in hex string
-# Example: "FFFFFFFF"
-string sender_bsm_id
```

Comment:
BSM id should not be removed. It may not be used in IHP, but it is used in other plugins and use cases.

Reply:
Understand. I  put it back and re-compiled. Please check the new commit.